additional_rule_metadata.json

```diff
@@ -688,6 +719,14 @@
         "recommendation": "false, // enable this rule only if you are legally required to add a file header",
         "level": "Opportunity for Excellence"
     },
+    "function-constructor": {
+        "issueClass": "Non-SDL",
```

Comment:
This metadata drastically differs from initial [rule metadatedata](https://github.com/Microsoft/tslint-microsoft-contrib/blob/9b25025c4912de65ca869c93a37c8bec84c94111/src/noFunctionConstructorWithStringArgsRule.ts#L15-L20)

Is it intentional change?

Reply:
No, copy & paste error on my end - will fix up all of these...